src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -29,4 +34,50 @@ public static String convertToJsonUsingGson(Object object) {
     Gson gson = new Gson();
     return gson.toJson(object);
   }
+
+  /*
+   * Class that returns the ID of a logged in user.
+   */
+  public static int getUserId() {
+    int userId = -1;
+
+    // Get logged in user email.
+    UserService userService = UserServiceFactory.getUserService();
+    String email = userService.getCurrentUser().getEmail();
+
+    // If user is not logged in, return -1.
+    if (email = "") {
+      return userId;
+    }
+
+    // Set up variables needed to connect to MySQL database.
+    String url = "jdbc:mysql://localhost:3306/Mintern?useSSL=false&serverTimezone=PST8PDT";
+    String user = "root";
+    String password = "";
```

Comment:
If you think that these will be used across various util functions, consider pulling these out to be constants - I assume these values will never change.

https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
Great idea! Done.